src/components/badge/badge.module.css

```diff
@@ -73,6 +73,10 @@
 ***
 */
 
+.base {
+	padding: 0;
```

Comment:
Since there isn't a container or outline with this style, the padding isn't needed.